pubspec.lock

```diff
@@ -0,0 +1,348 @@
+# Generated by pub
```

Comment:
can you add `pubspec.lock` to gitignore and remove from the PR. As `realtime-dart` is a library package, we don't need to commit that file.

Reply:
Sure.